radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/MainnetEngineRules.java

```diff
@@ -81,89 +81,86 @@
 import com.radixdlt.statecomputer.EpochProofVerifierV2;
 
 import java.util.Set;
+import java.util.function.Function;
 
-public enum RERulesVersion {
-	OLYMPIA_V1 {
-		@Override
-		public RERules create(RERulesConfig config) {
-			var maxRounds = config.getMaxRounds();
-			var perByteFee = config.getFeeTable().getPerByteFee().toSubunits();
-			var perResourceFee = config.getFeeTable().getPerResourceFee().toSubunits();
-			var rakeIncreaseDebouncerEpochLength = config.getRakeIncreaseDebouncerEpochLength();
+public final class MainnetEngineRules {
```

Comment:
Seems to me to be cleaner to have this be an `enum` as was implemented previously, since we want clear explicit identifiers to specific Ruleset. Any reason it's been changed?